src/components/tasks/TasksContent.tsx

```diff
@@ -143,40 +144,48 @@ export const TasksContent = ({ dev }: { dev?: boolean }) => {
-            >
-                <TabSection
-                    dev={dev}
-                    onSelect={(status: Tab) =>
-                        searchNewTasks(status, queryAssignees, queryTitle)
-                    }
-                    activeTab={selectedTab}
-                />
-            </div>
-            <div
-                className={classNames['status-select-container']}
-                data-testid="status-select-container"
-            >
-                <Select
-                    dev={dev}
-                    value={{
-                        label: getChangedStatusName(selectedTab),
-                        value: selectedTab,
-                    }}
-                    onChange={(selectedTaskStatus) => {
-                        if (selectedTaskStatus) {
-                            searchNewTasks(
-                                selectedTaskStatus.value as Tab,
-                                queryAssignees,
-                                queryTitle
-                            );
-                        }
-                    }}
-                    options={taskSelectOptions}
-                />
-            </div>
+            {dev === true ? (
+                <>
+                    <div
+                        className={classNames['status-tabs-container']}
+                        data-testid="status-tabs-container"
+                    >
+                        <TabSection
+                            dev={dev}
+                            onSelect={(status: Tab) =>
+                                searchNewTasks(
+                                    status,
+                                    queryAssignees,
+                                    queryTitle
+                                )
+                            }
+                            activeTab={selectedTab}
+                        />
+                    </div>
+                    <div
+                        className={classNames['status-select-container']}
+                        data-testid="status-select-container"
+                    >
+                        <Select
+                            dev={dev}
+                            value={{
+                                label: getChangedStatusName(selectedTab),
+                                value: selectedTab,
+                            }}
+                            onChange={(selectedTaskStatus) => {
+                                if (selectedTaskStatus) {
+                                    searchNewTasks(
+                                        selectedTaskStatus.value as Tab,
+                                        queryAssignees,
+                                        queryTitle
+                                    );
+                                }
+                            }}
+                            options={taskSelectOptions}
+                        />
+                    </div>
+                </>
+            ) : null}
```

Comment:
Donot understand when feature flag then you show tab??

Reply:
Thanks for point that, Have updated this we don't have to show this tab when feature flag is on